doc/en/user/source/styling/sld/extensions/rendering-transform.rst

```diff
@@ -408,3 +408,63 @@ and then the display of the NDVI index computed with the above style:
 
 .. figure:: images/s2-ndvi.png
    :align: center
+
+
+Group candidate selection
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``vec:GroupCandidateSelection`` is a **Vector-to-Vector** rendering transformation
+which filters a FeatureCollection according to the aggregate operation chosen (MIN or MAX) and the groups defined through attribute names. Given a feature collection, groups according to the defined grouping attributes, and returns the feature having the MIN or MAX value for the chosen attribute. One feature will be chosen for each group.
+The following SLD invokes the transformation:
+
+.. code-block:: xml
+
+   <?xml version="1.0" encoding="UTF-8"?>
+       <sld:StyledLayerDescriptor xmlns:sld="http://www.opengis.net/sld" xmlns="http://www.opengis.net/sld" xmlns:datex_3_0="http://www.vegvesen.no/datex/3.0" xmlns:gml="http://www.opengis.net/gml" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.0.0">
+        <sld:UserLayer>
+        <sld:UserStyle>
+            <sld:Name>Default Styler</sld:Name>
+            <sld:Title>Default Styler</sld:Title>
+            <sld:FeatureTypeStyle>
+                <Transformation>
+                    <ogc:Function name="vec:GroupCandidateSelection">
+                        <ogc:Function name="parameter">
+                            <ogc:Literal>data</ogc:Literal>
+                        </ogc:Function>
+                        <ogc:Function name="parameter">
+                            <ogc:Literal>operationAttribute</ogc:Literal>
+                            <ogc:Literal>datex_3_0:speed</ogc:Literal>

```

Comment:
This could be confusing to those not used to see complex features, could you add a comment that "datex_3_0:speed", "datex_3_0:highway" and "datex_3_0:lane" are attributes of a complex feature type? One might wonder if it's some sort of odd syntax to refer to a "layer:attribute" structure instead.